### PR TITLE
Added entries: Bzowski2018, Bzowski2019

### PR DESCRIPTION
**Pull request by kbzowski**

### CODE REVIEW

File: publications-2018.bib [hunk 1]
```diff
@@ -1,5 +1,17 @@
 % Encoding: US-ASCII
 
+@article{Bzowski2018,
+  title = {Application of statistical representation of the microstructure to modeling of phase transformations in DP steels by solution of the diffusion equation},
+  journal = {Procedia Manufacturing},
+  volume = {15},
+  pages = {1847--1855},
+  year = {2018},
+  note = {Proceedings of the 17th International Conference on Metal Forming METAL FORMING 2018 September 16 – 19, 2018, Loisir Hotel Toyohashi, Toyohashi, Japan},
```
Comment (masterleinad) [inline publications-2018.bib]:
```suggestion
  note = {Proceedings of the 17th International Conference on Metal Forming METAL FORMING 2018 September 16 -- 19, 2018, Loisir Hotel Toyohashi, Toyohashi, Japan},
```

File: publications-2019.bib [hunk 2]
```diff
@@ -1,5 +1,13 @@
 % Encoding: US-ASCII
 
+@phdthesis{Bzowski2019,
+  author       = {Krzysztof Bzowski},
```
Comment (bangerth) [inline publications-2019.bib]: Congratulations :-)

**Comment by bangerth** [comment]
I think you forgot to push the changed version.

File: publications-2019.bib (hunk 2 above)
Comment (kbzowski) [reply publications-2019.bib]: Thank you.  Without deal.ii it would be much harder.

**Comment by masterleinad** [comment]
I guess you have to rebase your branch onto `master`. The CI check seems to complain about unrelated entries.

**Comment by bangerth** [comment]
@kbzowski ?

**Comment by kbzowski** [comment]
Sorry for delay. What should I do now? I see "This pull request can be automatically merged by project collaborators"

**Comment by masterleinad** [comment]
We need the CI test to pass. Can you rebase on top of the current master branch?

**Comment by kbzowski** [comment]
Done. Should I create new PR?

**Comment by tjhei** [comment]
> Done. Should I create new PR?

No, just rebase to master and push to the same branch.

**Comment by tjhei** [comment]
Thanks! :+1:

**Pull request merged**